### PR TITLE
blog: What’s new in LocalStack for Snowflake?

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

**Review by eruditmorina — APPROVED** [review]
Look good to me!
Thanks @HarshCasper 🚀

File: content/2024-08-28-whats-new-in-localstack-for-snowflake/index.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+lead: What’s new in LocalStack for Snowflake?
+date: 2024-08-28T11:57:42+05:30
+lastmod: 2024-08-28T11:57:42+05:30
+images: []
+contributors: []
+tags: ['news']
+---
+
+## Introduction
+
```
Comment (whummer) [inline content/2024-08-28-whats-new-in-localstack-for-snowflake/index.md:13]: nit: I'm assuming this part is still missing? Let us know if you need any help here - happy to write up a few sentences for this paragraph.. 👍

File: content/2024-08-28-whats-new-in-localstack-for-snowflake/index.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+// picture of the new web UI
+
+The Web User Interface is still in **preview**, and we are actively working on adding new features and improving the user experience. Learn more about the new Web User Interface in our [documentation](https://snowflake.localstack.cloud/user-guide/web-user-interface/).
+
+### Support for LocalStack Ephemeral Instances
+
+We have launched Ephemeral Instances, enabling you to run a LocalStack for Snowflake sandbox in the cloud rather than on your local machine. This ephemeral environment is a short-lived, encapsulated deployment of the Snowflake emulator in the cloud. With these sandboxes, you can run tests, preview features in your Snowflake applications, and collaborate asynchronously within and across your team!
+
+// picture
```
Comment (whummer) [inline content/2024-08-28-whats-new-in-localstack-for-snowflake/index.md:90]: Wondering if we can get some help from @gtsiolis to brush up the design of the UI a bit. (but shouldn't be a blocker - as we have lots of other design work to prepare for product/re:Invent right now..)

File: content/2024-08-28-whats-new-in-localstack-for-snowflake/index.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+Snowflake Dynamic Tables allow a background process to continuously load new data from sources into the table, accommodating both delta and full load operations. These tables automatically update to reflect query results, eliminating the need for a separate target table and custom data transformation code. They are regularly updated through scheduled refreshes by an automated process.
+
+LocalStack for Snowflake now supports Dynamic tables, enabling you to create and manage them locally. The supported SQL statements for managing Dynamic tables include:
+
+* [`CREATE DYNAMIC TABLE`](https://docs.snowflake.com/en/sql-reference/sql/create-dynamic-table.html)
+* [`DROP DYNAMIC TABLE`](https://docs.snowflake.com/en/sql-reference/sql/drop-dynamic-table.html)
+
+For more detailed information on using Dynamic tables, refer to our [documentation](https://snowflake.localstack.cloud/user-guide/dynamic-tables/).
+
+## Conclusion
```
Comment (whummer) [inline content/2024-08-28-whats-new-in-localstack-for-snowflake/index.md:148]: Again, happy to provide a few sentences here as a starting point, to round up the blog, and provide some outlook for future roadmap.. 👍

**Review by whummer — COMMENTED** [review]
Looks great, kudos for tackling this @HarshCasper ! 🚀 Perhaps we could also mention some of the integrations we've recently built (e.g., see samples [here](https://github.com/localstack-samples/localstack-snowflake-samples/)). Happy to contribute to this as we go .. 👍

File: content/2024-08-28-whats-new-in-localstack-for-snowflake/index.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+We have released a new Web User Interface for LocalStack for Snowflake. This interface is accessible via a web browser and provides a user-friendly experience for managing your local Snowflake resources. The interface includes a dashboard that allows you to:
+
+-   Run Snowflake SQL queries and view the results using a Query Editor.
+-   View detailed request/response traces of API calls made to Snowflake.
+-   Forward queries from the Snowflake emulator to a real Snowflake instance using a proxy.
+
+You can access the Web User Interface by navigating to [https://snowflake.localhost.localstack.cloud](https://snowflake.localhost.localstack.cloud) in your web browser.
+
+// picture of the new web UI
```
Comment (hovaesco) [inline content/2024-08-28-whats-new-in-localstack-for-snowflake/index.md:82]: missing picture or will it be added in the next iteration?

File: content/2024-08-28-whats-new-in-localstack-for-snowflake/index.md (hunk 2 above)
Comment (gtsiolis) [reply content/2024-08-28-whats-new-in-localstack-for-snowflake/index.md:90]: To clarify, we're referring to the instance creation flow step, right? Cc @whummer 

If so, just created a [relevant task](https://www.notion.so/localstack/f3974f892ccf4e3ea57bfc7a00b62e01) (internal) to keep track of this.

| New instance w/ subscription | New instance w/o subscription | 
|-|-|
| Frame 48096240 | Frame 48096241 |

Comment (whummer) [reply content/2024-08-28-whats-new-in-localstack-for-snowflake/index.md:90]: Ah sorry @gtsiolis for not being clear, I was more referring to the internal UI that we have in the Snowflake emulator itself (https://snowflake.localhost.localstack.cloud , only accessible if it is running locally). 

The ephemeral instances UI in the Web app looks fine, 👍 no need to improve the design of it at this point.. :)

But again, we currently have other priorities, so I don't see us spending cycles on the Snowflake UI improvements at this point. Probably post re:Invent!

Comment (gtsiolis) [reply content/2024-08-28-whats-new-in-localstack-for-snowflake/index.md:90]: Thanks for clarifying, @whummer! 😃

I recently opened a [draft task](https://www.notion.so/localstack/10cfc2a2343180b798bdc98f68cf2347?pvs=4) (internal) for breaking down things around the Snowflake UI along with an the editor within the web app later on, see also [relevant discussion](https://localstack-cloud.slack.com/archives/C06CV9US815/p1727276477705329) (internal). 🏓

**Comment by hovaesco** [comment]
@HarshCasper please include latest additions from `v0.2.3` release https://github.com/localstack/snowflake-docs/pull/49

**Pull request closed without merging**